index.html

```diff
@@ -1578,7 +1578,7 @@ <h1>Advanced Concepts</h1>
     describe such value typing.</p></dd>
   <dt>Value ordering</dt>
   <dd><p>In JSON, a property with an array value implies an implicit order;
-    arrays in JSON-LD do not provide an ordering of the contained elements by
+    arrays in JSON-LD do not convey any ordering of the contained elements by
```

Comment:
It might just be me, being not a native speaker,
but as I see it, an array always *provide* an ordering.
It is just that we decide to to ignore it (unless they appear as the value of `@list`).

Reply:
I think both wordings are fairly close, and it comes down to editor's choice.